phoenix-core/src/main/java/org/apache/phoenix/mapreduce/AbstractBulkLoadTool.java

```diff
@@ -226,6 +228,14 @@ private int loadData(Configuration conf, CommandLine cmdLine) throws Exception {
         configureOptions(cmdLine, importColumns, conf);
         String sName = SchemaUtil.normalizeIdentifier(schemaName);
         String tName = SchemaUtil.normalizeIdentifier(tableName);
+
+        // There must be a better solution for this
```

Comment:
nit:
I don't think there really is.
We could query the table via HBase API directly, but that wouldn't really help.